### ISSUE TITLE
signinPopup open window not centered

### ISSUE THREAD

**Issue by l0kers** [authts/oidc-client-ts#799]
Call UserManager.signinPopup method open new window but not centered in front of the current window. This issue occurs when user has multiple screens. 
Method PopupUtils.center works perfect on main screen, but if application is run on an additional screen a new window is opened on the side of the screen. 
On additional screen 'window.screenX' can have value eg: -1400.

**Comment by pamapa** [comment]
Have you seen https://authts.github.io/oidc-client-ts/interfaces/UserManagerSettings.html#popupWindowFeatures?

**Comment by l0kers** [comment]
Yes, I saw it. I know that I can calculate it by myself but I just want to let you know that the PopupUtils.center method doesn't work correctly... not every time.
